atlp-rwanda/rca-barefoot-team6-bn
# chore:.env setup and added Hound badge

**What does this PR do?**
.env setup and attaches Hound badge the readme

**Description of Task to be completed?**
Hound badge is displayed in the readme file and you can use environment variables

**How should this be manually tested?**
you can declare a variable  in .env file then try accessing it by process.env.variable 
To see the hound badge, it's displayed in the readme file on GitHub

**What are the relevant pivot tracker stories** 
#184638362  #184639260
